x-pack/test/security_solution_endpoint/apps/endpoint/responder.ts

```diff
@@ -34,7 +34,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     await pageObjects.responder.closeResponder();
   };
 
-  describe('Response Actions Responder', function () {
+  describe.skip('Response Actions Responder', function () {
```

Comment:
i believe i will be replacing this with some unit tests since the "capabilities" part of the endpoint metadata is more easily manipulated with jest tests